index.html

```diff
@@ -53,6 +53,49 @@
     <div class="page-content">
         <p>This is the page.</p>
     </div>
+    <byu-footer>
+        <byu-footer-column class="triple-wide">
```

Comment:
We may want to remove the "triple-wide" class from this test page and use a separate documentation page for double- and triple-wide columns.